test/cookbooks/app_test/libraries/helpers.rb

```diff
@@ -0,0 +1,17 @@
+module OslAppTest
+  module Cookbook
+    module Helpers
+      def get_return_code(site)
```

Comment:
I'm getting the following similar issue on both app2 and app3 that I'm not sure if it's related to this method or not:
```
       Still waiting for redmine.replicant.us to start... 155 seconds
       
           
           ================================================================================
           Error executing action `run` on resource 'ruby_block[wait_for_replicant]'
           ================================================================================
           
           RuntimeError
           ------------
           Failed to start redmine.replicant.us
```

Reply:
A second converge passes, so I suspect the sites are just taking longer to start up. I believe this was happening with the old method as well.